src/main/java/bio/terra/cli/command/app/passthrough/Gcloud.java

```
@@ -15,4 +37,49 @@ public String getExecutableName() {
   public String getInstallationUrl() {
     return "https://cloud.google.com/sdk/docs/install";
   }
+
+  @Override
+  protected void executeImpl() {
+    workspaceOption.overrideIfSpecified();
+
+    if (bucketName != null && builds && submit) {
```

If GCS bucket isn't set, you should throw an error.

```
if builds submit {
  if GCS bucket isn't set
    throw error
  Now you can assume GCS bucket is set, and proceed.
```
```